x/datadeal/keeper/deal_test.go

```diff
@@ -183,7 +183,7 @@ func (suite *dealTestSuite) TestSellDataSuccess() {
 	suite.Require().Equal(dataSale.DataHash, suite.dataHash1)
 }
 
-func (suite *dealTestSuite) TestSellDataStatusFailed() {
+func (suite *dealTestSuite) TestReSellData() {
```

Comment:
Good.
I think it would be good to check the desired events are emitted.

Reply:
Fixed in [9be3da9](https://github.com/medibloc/panacea-core/pull/486/commits/9be3da9de8b7d7787d0cecf1a4c8d022d5dd02b3).